### PR TITLE
Fix: input focus bug

### PR DESCRIPTION
**Pull request by dleard**
- input focus was caused by moving the CUSTOM_FIELDS in Form.tsx inside the function component in an attempt to not rely on formContext. I've undone this change & we can revisit the idea of not using formContext later
- fixed the dev data in formResult.sql, numeric values were quoted. This was causing them to be parsed as strings & throw validation errors
- added a null diffQuery prop to the ApplicationWizardConfirmation file to remove a relay warning
- removed format: naics from the administration form_json as it is unused now that we are using a dropdown. (was throwing a relay warning)
- fixed the dev data in application-2018.sql fixture to not rely on hardcoded application IDs

### CODE REVIEW

**Review by pbastia — CHANGES_REQUESTED** [review]
Just a `console.log` left behind, looks great!

**Review by kriscooke — COMMENTED** [review]
👌 Thanks, looks to to me!

**Review by pbastia — DISMISSED** [review]

File: schema/data/dev/form_result.sql [hunk 1]
```diff
@@ -52,7 +52,7 @@ begin;
...
...nnualEmission": 90, "gasDescription": "Carbon dioxide from biomass not listed in Schedule C of GGERR"}, {"gwp": 1, "gasType": "bioCO2", "annualCO2e": 5, "annualEmission": 5, "gasDescription": "Carbon dioxide from biomass listed in Schedule C of GGERR"}, {"gwp": 25, "gasType": "CH4", "annualCO2e": 8000, "annualEmission": 320, "gasDescription": "Methane"}, {"gwp": 298, "gasType": "N20", "annualCO2e": 98340, "annualEmission": 330, "gasDescription": "Nitrous oxide"}], "sourceTypeName": "Waste"}, {"gases": [{"gwp": 1, "gasType": "CO2nonbio", "annualCO2e": 340, "annualEmission": 340, "gasDescription": "Carbon dioxide from non-biomass"}, {"gwp": 1, "gasType": "CO2", "annualCO2e": 350, "annualEmission": 350, "gasDescription": "Carbon dioxide from biomass not listed in Schedule C of GGERR"}, {"gwp": 1, "gasType": "bioCO2", "annualCO2e": 360, "annualEmission": 360, "gasDescription": "Carbon dioxide from biomass listed in Schedule C of GGERR"}, {"gwp": 25, "gasType": "CH4", "annualCO2e": 9250, "annualEmission": 370, "gasDescription": "Methane"}, {"gwp": 298, "gasType": "N20", "annualCO2e": 113240, "annualEmission": 380, "gasDescription": "Nitrous oxide"}], "sourceTypeName": "Wastewater"}]}'),
```
Comment (matthieu-foucault) [inline schema/data/dev/form_result.sql:55]: same issue L19 above. Those lines should be formatted for better readability

File: schema/data/dev/application-2018.sql [hunk 2]
```diff
@@ -22,17 +22,25 @@ begin;
...
-  -- Set legal_disclaimer_accepted to true for application id=3
-  update ggircs_portal.application_revision set legal_disclaimer_accepted = true where application_id=3;
+  do $$
+    declare app_id integer;
+    begin
+      app_id = (select id from ggircs_portal.application where reporting_year=2018 and facility_id=2);
+
+      -- Set application submitted
+      update ggircs_portal.application_revision_status set application_revision_status='submitted' where application_id=app_id and version_number=1;
+      -- Create a legacy application_review_step for the 2018 application with ID=3
```
Comment (matthieu-foucault) [inline schema/data/dev/application-2018.sql:32]: just some minor updates needed in the comments (`ID=3`)

**Review by matthieu-foucault — CHANGES_REQUESTED** [review]

**Review by matthieu-foucault — APPROVED** [review]

**Pull request merged**